theano/gof/cmodule.py

```diff
@@ -1711,8 +1711,8 @@ class Compiler(object):
 
     """
 
-    @staticmethod
-    def _try_compile_tmp(src_code, tmp_prefix='', flags=(),
+    @classmethod
+    def _try_compile_tmp(cls, src_code, tmp_prefix='', flags=(),
                          try_run=False, output=False, compiler=None):
         """
         Try to compile (and run) a test program.
```

Comment:
Add in all changed method the docstring to tell that we prepend to the flags the compile_args flags